docker/ecco-java.security

```diff
@@ -0,0 +1,3 @@
+jdk.tls.disabledAlgorithms=SSLv3, RC4, DES, MD5withRSA, DH keySize < 1024, \
+    EC keySize < 224, 3DES_EDE_CBC, anon, NULL, \
+    include jdk.disabled.namedCurves
```

Comment:
I think disabling TLS 1.0 is appropriate too.

Not sure why "my" `DH keySize` was on the next line though (I modified my local Java8 JRE file to get to this suggestion) but seeing it makes the lines a but more equal in length I think it's fine. :)

```suggestion
jdk.tls.disabledAlgorithms=SSLv3, TLSv1, RC4, DES, MD5withRSA, \
    DH keySize < 1024, EC keySize < 224, 3DES_EDE_CBC, anon, NULL, \
    include jdk.disabled.namedCurves
```

Reply:
f3275bb